packages/mds-audit/tests/api.spec.ts

```diff
@@ -84,13 +85,33 @@ describe('Testing API', () => {
       db.writeEvent({
         provider_id,
         device_id: provider_device_id,
-        event_type: 'trip_start',
-        telemetry: {
+        event_type: 'agency_drop_off',
```

Comment:
This isn't an event_type that's valid from the perspective of `agency` (which is how we're ingesting all of the vehicle data audit is running against). Intentional?

Reply:
Not intentional - I'd like a realistic `event_type` here. I just pulled this from `mds-types.VEHICLE_EVENTS`. Is `'trip_start'` one that `agency` uses?

Reply:
Good call